exporters/committime/collector_azure_devops.py

```diff
@@ -39,13 +39,17 @@ def get_commit_time(self, metric: CommitMetric):
                 "Skipping non Azure DevOps server, found %s" % (git_server)
             )
         logging.debug("metric.repo_project %s" % (metric.repo_project))
-        logging.debug("metric.git_api %s", self.git_api)  # TODO: metric, not self
+        logging.debug("metric.git_server %s" % (metric.git_server))
 
         # Fill in with your personal access token and org URL
         # personal_access_token = 'YOURPAT'
         # organization_url = 'https://dev.azure.com/YOURORG'
         personal_access_token = self.token
-        organization_url = self.git_api.url
+        organization_url = (
+            self.git_api.url + "/" + metric.repo_group
```

Comment:
It seems unnecessary to me to have the [GIT_API](https://pelorus.readthedocs.io/en/latest/GettingStarted/configuration/ExporterCommittime/#git_api) option, as we get it from the metric.

Can I open an issue to remove it?

Reply:
hrm.. is that true for all build types? Sure.. open an issue but let's discuss in the issue.